server/crawler/starlark_docstring_parser_test.go

```diff
@@ -91,3 +91,16 @@ func TestDocstringParser_EmptyDescription(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, result)
 }
+
+func TestDocstringParser_NotParameterizedDict(t *testing.T) {
+	docstring := `
+    Args:
+		string_argument (string): A simple string argument.
+		wrong_dict (dict[x, x]): A not parameterized dictionary example. 
```

Comment:
@leoporoli could we also add a test that checks for unparameterized (this was the original issue):

```
		wrong_dict (dict): A not parameterized dictionary example. 
```

Reply:
sure